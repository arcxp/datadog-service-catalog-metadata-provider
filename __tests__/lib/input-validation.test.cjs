const { validateDatadogHostname } = require('../../lib/input-validation')

describe('validateDatadogHostname()', () => {
  test('valid hostname', () => {
    expect(validateDatadogHostname('app.datadoghq.com')).toBe(
      'app.datadoghq.com',
    )
  })
})
