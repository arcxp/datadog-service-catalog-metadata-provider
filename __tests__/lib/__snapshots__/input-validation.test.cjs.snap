// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`validateDatadogHostname() invalid hostnames 1`] = `"Invalid DataDog host: somethingcompletelydifferent.com. See here for more details: https://docs.datadoghq.com/getting_started/site/"`;
